projects/kubernetes/autoscaler/docker/linux/Dockerfile

```diff
@@ -0,0 +1,14 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base
+FROM $BASE_IMAGE
+
+ARG RELEASE_BRANCH
+ARG TARGETARCH
+ARG TARGETOS
+
+COPY _output/$RELEASE_BRANCH/bin/autoscaler/$TARGETOS-$TARGETARCH/cluster-autoscaler-amd64 /cluster-autoscaler
+COPY _output/$RELEASE_BRANCH/LICENSES /LICENSES
+COPY $RELEASE_BRANCH/ATTRIBUTION.txt /ATTRIBUTION.txt
+
+USER nobody
```

Comment:
this should be fine, but keep in mind in case we get issues when we test the image